src/templates/hl7v2/Resources/Location.hbs

```diff
@@ -3,16 +3,21 @@
     "resource":{
         "resourceType": "Location",
         "id":"{{ID}}",
-        {{>DataType/PL.hbs PL=PL}},
-        {{>DataType/XONLocation.hbs XON=ORC-21}},
+        {{#each (getFieldRepeats PL)}}
```

Comment:
Please check the FHIR specification. The fields getting generated by PL.hbs cannot be repeated.